src/me/StevenLawson/TotalFreedomMod/Commands/Command_setl.java

```diff
@@ -7,7 +7,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Sets everyone's Worldedit block modification limit to 500.", usage = "/<command>")
+@CommandParameters(description = "Sets everyone's Worldedit block modification limit to 500.", usage = "/<command>", aliases = "setlimit")
```

Comment:
why do you want to waste your time to type /setlimit while you could have just type /setl, its faster..